docs/source/guides/converting-to-openapi.rst

```diff
@@ -196,6 +196,27 @@ openapi.use.xml (boolean)
     Enables converting Smithy XML traits to OpenAPI XML properties. (this
     feature is not yet implemented).
 
+openapi.use.convertedExternalDocs ([string])
```

Comment:
I'm going to be renaming these anyways..., but I'd probably just call this `openapi.externalDocs`.